library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/BaseWireframeMapper.kt

```diff
@@ -12,16 +12,24 @@ import android.graphics.drawable.InsetDrawable
 import android.graphics.drawable.RippleDrawable
 import android.os.Build
 import android.view.View
+import com.datadog.android.sessionreplay.internal.AsyncImageLoadingCallback
 import com.datadog.android.sessionreplay.internal.recorder.GlobalBounds
+import com.datadog.android.sessionreplay.internal.recorder.ViewUtilsInternal
+import com.datadog.android.sessionreplay.internal.recorder.base64.Base64Serializer
+import com.datadog.android.sessionreplay.internal.recorder.base64.Base64SerializerFactory
+import com.datadog.android.sessionreplay.internal.utils.Base64Utils
 import com.datadog.android.sessionreplay.model.MobileSegment
 import com.datadog.android.sessionreplay.utils.StringUtils
 import com.datadog.android.sessionreplay.utils.ViewUtils
 
 @Suppress("UndocumentedPublicClass")
 abstract class BaseWireframeMapper<T : View, S : MobileSegment.Wireframe>(
     private val stringUtils: StringUtils = StringUtils,
-    private val viewUtils: ViewUtils = ViewUtils
-) : WireframeMapper<T, S> {
+    private val viewUtils: ViewUtils = ViewUtils,
+    protected val base64Utils: Base64Utils = Base64Utils,
+    protected val base64Serializer: Base64Serializer = Base64SerializerFactory.build(),
+    protected val viewUtilsInternal: ViewUtilsInternal = ViewUtilsInternal()
```

Comment:
Should refactor to remove viewUtilsInternal - perhaps by moving the method into viewUtils